next.config.js

```diff
@@ -22,36 +22,41 @@ const webpack = (config) => {
 }
 
 module.exports = (phase) => {
```

Comment:
whats the benefit to rewriting the next config in this way?

Reply:
There are a few variables you want to share between development and production environments. It's a bit more neat to use object spread for conditionally adding environment-specific variables than to duplicate all the other ones